.gitignore

```diff
@@ -29,51 +29,12 @@ logs
 *.log
 npm-debug.log*
```

Comment:
Can you delete lines https://github.com/kingsleyzissou/breakpoint.css/blob/2d8663ccff82530aef09496006257c3502b93d9c/.gitignore#L3-L30 
and .npm